Tests/MessagingInApp/Views/InAppMessageViewTest.swift

```diff
@@ -168,51 +182,43 @@ class InAppMessageViewTest: UnitTest {
         let inlineView = InAppMessageView(elementId: givenOldInlineMessage.elementId!)
         let webViewBeforeFetch = getInAppMessageWebView(fromInlineView: inlineView)
 
-        // Make sure message is unique, but has same elementId.
-        let givenNewInlineMessage = Message(messageId: .random, campaignId: .random, elementId: givenOldInlineMessage.elementId)
+        // Make sure message is a new message, but has same elementId.
+        let givenNewInlineMessage = Message(queueId: .random, elementId: givenOldInlineMessage.elementId)
 
         await simulateSdkFetchedMessages([givenNewInlineMessage])
 
         let webViewAfterFetch = getInAppMessageWebView(fromInlineView: inlineView)
 
         // If the WebViews are different, it means the message was reloaded.
         XCTAssertTrue(webViewBeforeFetch === webViewAfterFetch)
+        XCTAssertEqual(getInAppMessage(forView: inlineView), givenOldInlineMessage)
     }
 
     // MARK: expiration of in-app messages
 
     @MainActor
-    func test_expiration_givenDisplayedMessageExpires_expectDismissView() async {
+    func test_expiration_givenDisplayedMessageExpires_expectContinueShowingMessageUntilClose() async {
```

Comment:
While this PR focuses on replacing an in-app message with another, we expect that a message is replaced only after the close button has been clicked. 

That means that this test function needed modified because the expected behavior has changed for messages that expire.